src/models.rs

```diff
@@ -286,17 +278,11 @@ impl BlockId {
     pub fn id(&self) -> &str {
         &self.0
     }
-
-    pub fn from(page_id: &PageId) -> Self {
-        BlockId(page_id.clone().0)
-    }
 }
 
-impl Identifiable for BlockId {
-    type Type = BlockId;
-
-    fn id(&self) -> &Self::Type {
-        self
+impl From<PageId> for BlockId {
```

Comment:
do we need this now that you can identify a `BlockId` directly from the `Page` object?